### ISSUE TITLE
[Improvement]: Deprecation message for passing null to strlen

### ISSUE THREAD

**Issue by SpicySpaceman** [pimcore/pimcore#14795]
### Improvement description

Seeing `Deprecated: strlen(): Passing null to parameter #1 ($string) of type string is deprecated` in the logs:

https://github.com/pimcore/pimcore/blob/4bccaed3f2a2aebee1a6f1655199c76d3efa2762/models/User.php#L589-L596

**Comment by dvesh3** [comment]
@danidoedel Is it really triggered from this place? as the `strlen` check doesn't call this in case of null.

**Comment by SpicySpaceman** [comment]
> @danidoedel Is it really triggered from this place? as the `strlen` check doesn't call this in case of null.

@dvesh3 That's the complete log entry:

```
[2023-03-30T12:21:31.316164+04:00] php.INFO: Deprecated: strlen(): Passing null to parameter #1 ($string) of type string is deprecated {"exception":"[object] (ErrorException(code: 0): Deprecated: strlen(): Passing null to parameter #1 ($string) of type string is deprecated at /var/www/html/vendor/pimcore/pimcore/models/User.php:591)"} []
```

So yeah I am pretty confident that this is the correct place. Unless I am completely confusing something 😄

**Comment by dvesh3** [comment]
you are right...I was looking at explode 🤦‍♂️ thinking it is throwing exception. thanks for confirming 😄

**Comment by blankse** [comment]
Is already merged and in release 10.5.21, right?

**Comment by mcop1** [comment]
> Is already merged and in release 10.5.21, right?

Yes, merged already and the change is in the 10.5.21 release, but the issue wasn´t closed back then. Thanks!
